send_email.py

```diff
@@ -0,0 +1,37 @@
+import smtplib, ssl
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+
+sender_from_email = "sender@gmail.com"
+receiver_to_email = "receiver@gmail.com"
+password = input("Type your password and hit enter:")
```

Comment:
Could you please change the input to getpasswd - I think this might be better because you cannot see the direkt input.

Reply:
yes changed the implementation to getpass()